src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,19 @@
+package duke.commands;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.Task;
+import duke.TaskList;
+import duke.Ui;
+
+public class Command {
+
+    public void execute(TaskList list, Storage store, Ui ui) throws DukeException {
```

Comment:
Maybe you can consider making this method abstract and making the `Command` class abstract.